source/layouts/article_layout.haml

```diff
@@ -0,0 +1,42 @@
+:ruby
+  site_content_server = data.site.content_server_url
+  -# site_asset_server = data.site.asset_server_url
+  article_title = current_article.data.title
+  article_author = current_article.data.author
+  article_date = current_article.date
+  article_image = current_article.data.image.to_s
+  article_image_lg = current_article.data.image_lg.to_s
+  article_caption = current_article.data.caption.to_s
+
+= wrap_layout :layout do
+  - content_for :meta_desc do
+    = tag "meta", name: "description", content: article_title
+
+  %article
+    %section.container
+      .article-title
+        %h1= article_title
```

Comment:
Need to add some media queries to handle scaling down text for mobile screens.

![screenshot 2017-09-21 11 25 40](https://user-images.githubusercontent.com/615841/30686073-c1a8ba02-9ebf-11e7-95d7-57ad9a180e7f.png)

As an example, here's how I handle it on my blog: https://github.com/joshukraine/ofreport.com/blob/6bb315b07b6a69290d718caf5cd4d4fe9b31b5de/source/assets/stylesheets/patterns/_container.scss#L31